examples/03-ui-components/14-experimental-mobile-formatting-toolbar/README.md

```diff
@@ -0,0 +1,10 @@
+# Experimental Mobile Formatting Toolbar
+
+This example shows how to use the experimental mobile formatting toolbar, which uses [Visual Viewport API](https://developer.mozilla.org/en-US/docs/Web/API/Visual_Viewport_API) to position the toolbar right above the virtual keyboard on mobile devices.
+
+Controller is currently marked **experimental** due to the flickering issue with positioning (caused by delays of the Visual Viewport API)
```

Comment:
When is the flickering visible? I don't think I saw it when playing around in the preview

Reply:
Did you test on mobile? When you scroll / down a large document the updating of the position is a bit delayed